setup-env.template

```diff
@@ -12,6 +12,7 @@ export USE_CENTRAL_ECR=false
 export ADMIN_ROLENAME=aws-admin-role
 export DEPLOYMENT_NAME=cluster-name
 export ENVIRONMENT=sandbox  # sandbox, dev, test, or prod
+export DD_API_KEY_ID=look-in-aws-ssm
```

Comment:
since usually you export DATADOG_API_KEY and DATADOG_APP_KEY maybe we should keep it the same even though it's a long 4 word variable

Reply:
I don't think we're actually even setting an app key.  At least Fox hasn't been doing so.

Reply:
For reference, this is what he sent me: https://innerspace.stsci.edu/pages/viewpage.action?spaceKey=DMD&title=DataDog+setup.

We can definitely add more later, but the criteria to be met here is simply to replace his hardcoded API Key with one that is fetched on-the-fly with proper credentials.

Reply:
I meant name it DATADOG_API_KEY_ID